### PR TITLE
add troubleshooting

### PR DESCRIPTION
**Pull request by wrfly**
### Proposed changes

See: https://github.com/moby/moby/issues/34343

### Related issues

Fixes #34343

### CODE REVIEW

File: engine/userguide/storagedriver/device-mapper-driver.md [hunk 1]
```diff
@@ -623,6 +623,16 @@ block device and other parameters to suit your situation.
     <output truncated>
     ```
 
+### Active the `devicemapper` after reboot
```
Comment (JimGalasyn) [inline engine/userguide/storagedriver/device-mapper-driver.md]: I think you mean "activate".

File: engine/userguide/storagedriver/device-mapper-driver.md [hunk 2]
```diff
@@ -623,6 +623,16 @@ block device and other parameters to suit your situation.
...
+
+If you reboot the host and found that docker service failed to start,
+an error shows `Non existing device docker-thinpool`, you need to re-active
```
Comment (JimGalasyn) [inline engine/userguide/storagedriver/device-mapper-driver.md]: I think you mean "re-activate". Otherwise, LGTM.

Comment (wrfly) [reply engine/userguide/storagedriver/device-mapper-driver.md]: Thank you for your suggesting, my poor English....

**Comment by JimGalasyn** [comment]
Great, thank you for the contribution!

**Pull request merged**